cmd/imagec/hook.go

```diff
@@ -16,25 +16,25 @@ package main
 
 import (
 	"fmt"
-	"os"
+	"io"
 
 	"github.com/Sirupsen/logrus"
 )
 
 // ErrorHook is a simple logrus hook to duplicate Fatalf leveled error messages to stderr
 type ErrorHook struct {
```

Comment:
We can promote it to a pkg if someone else decides to use it, for now it's only used by imagec